gamebot/Program.cs

```diff
@@ -224,20 +235,22 @@ public void Start()
 				});
 			_client.Ready += (s,e) =>
```

Comment:
I'll move that code into the class's ctor because it don't think it references anything that would require the DAPI to be authenticated.